### ISSUE TITLE
若要进行fine-tuning，如何着手修改

### ISSUE THREAD

**Issue by April4lu** [argusswift/YOLOv4-pytorch#133]
>背景：采用了yolov4作为预训练模型
>目的：训练过程中不更新backbone的参数，仅更新其他部分的参数

仅修改train.py中的self.optimizer中传入的参数可以么？

![image](https://user-images.githubusercontent.com/32457231/104692626-3cd5c500-5743-11eb-864c-883a3f87a233.png)

**Comment by jingtianyilong** [comment]
Fine tune basically requires:
- current best checkpoint
- a new optimizer
- a new scheduler
For instance, my repo have fine-tune function build-in. You can also try out.
https://github.com/jingtianyilong/YOLOv4-pytorch
The Readme provide information for you to start training. With some minor modification, you can also use your current best checkpoint as a starting point.

**Comment by argusswift** [comment]
> > 背景：采用了yolov4作为预训练模型
> > 目的：训练过程中不更新backbone的参数，仅更新其他部分的参数
> 
> 仅修改train.py中的self.optimizer中传入的参数可以么？
> 
> ![image](https://user-images.githubusercontent.com/32457231/104692626-3cd5c500-5743-11eb-864c-883a3f87a233.png)

还需要指定部分冻结参数。eg:
for param in self.yolov4..backbone.parameters():
            param.requires_grad = False

**Comment by argusswift** [comment]
> > 背景：采用了yolov4作为预训练模型
> > 目的：训练过程中不更新backbone的参数，仅更新其他部分的参数
> 
> 仅修改train.py中的self.optimizer中传入的参数可以么？
> 
> ![image](https://user-images.githubusercontent.com/32457231/104692626-3cd5c500-5743-11eb-864c-883a3f87a233.png)

还需要指定部分冻结参数。eg:
for param in self.yolov4.backbone.parameters():
param.requires_grad = False

**Comment by argusswift** [comment]
> Fine tune basically requires:
> 
> * current best checkpoint
> * a new optimizer
> * a new scheduler
> 
> For instance, my repo have fine-tune function build-in. You can also try out.
> https://github.com/jingtianyilong/YOLOv4-pytorch
> The Readme provide information for you to start training. With some minor modification, you can also use your current best checkpoint as a starting point.

nice work
